TicTacToeApp/Game.swift

```diff
@@ -0,0 +1,36 @@
+
+public class Game {
+
+  private var board: Board
+  private var currentPlayer: PlayerMark
+
+  public init() {
+    board = Board()
+    currentPlayer = PlayerMark.X
+  }
+
+  public func makeMove(boardSpace: Int) {
+    board = board.markSpace(boardSpace, mark: currentPlayer)
+    currentPlayer = otherPlayer()
+  }
+
+  public func getCurrentPlayer() -> PlayerMark {
+    return currentPlayer
+  }
+
+  public func getInactivePlayer() -> PlayerMark {
+    return otherPlayer()
+  }
+
+  public func playerWonLastTurn(mark: PlayerMark) -> Bool {
```

Comment:
It does come off as confusing. I meant for it to be more descriptive in the context in which it is used in ViewController. I'll rename it to playerWon()